bip-0340.mediawiki

```diff
@@ -284,7 +285,8 @@ Blind Schnorr signatures could for example be used in [https://github.com/Elemen
 
 == Test Vectors and Reference Code ==
 
-For development and testing purposes, we provide a [[bip-0340/test-vectors.csv|collection of test vectors in CSV format]] and a naive, highly inefficient, and non-constant time [[bip-0340/reference.py|pure Python 3.7 reference implementation of the signing and verification algorithm]].
+For development and testing purposes, we provide a [[bip-0340/test-vectors.csv|collection of test vectors in CSV format]] and a naive, highly inefficient, and non-constant time [[bip-0340/reference.py|pure Python 3.7 reference implementation of the signing and verification algorithm]],
+both licensed under either the BSD-2-Clause License, the MIT License, or CC0 1.0, at your choice.
```

Comment:
"either … or" is only ever used with two options. May I suggest:
```suggestion
both licensed under one of the BSD-2-Clause License, the MIT License, or CC0 1.0, at your choice.
```

Reply:
> "either … or" is only ever used with two options.

That's obviously wrong. My usage is a counterexample.

On a more serious note, even experts disagree on this matter, see  https://en.wiktionary.org/wiki/either#cite_note-Pullum-2024-2 and https://english.stackexchange.com/questions/170409/whats-the-best-way-to-use-either-on-more-than-two-options?noredirect=1&lq=1#comment354637_170409 ... But yeah, let me make that change.

Reply:
Just dropped the "either" because "at your choice" should be clear enough.

Reply:
Fair enough, it sounded wrong to me and my native speaker colleagues collectively confirmed my stance, but of course only a Sith deals in absolutes.

![image](https://github.com/user-attachments/assets/a86d55af-3492-4cb4-906e-8f64ec21c3de)